test/server.test.ts

```diff
@@ -3,12 +3,15 @@
 // Licensed under the BSD-3-Clause license found in the LICENSE file or
 // at https://opensource.org/licenses/BSD-3-Clause
 
-import { Elt, Group, OPRFClient, OPRFServer, Oprf, Scalar, randomPrivateKey } from '../src/index.js'
+import { Oprf, OPRFClient, OPRFServer, randomPrivateKey } from '../src/index.js'
 
 import { jest } from '@jest/globals'
+import { GroupConsSjcl } from '../src/groupSjcl'
 
 const { sign, importKey } = crypto.subtle
 
+Oprf.Group = GroupConsSjcl
```

Comment:
Perhaps this could be done transparently behind an 

`import '@cloudflare/voprf-ts/scjl'`
or 
`import '@cloudflare/voprf-ts/noble'`

which could be declared as optional dependencies ? 

at some point when cloudflare is comfortable with noble, start recommending it ?

Reply:
On 2nd thought(or feel), perhaps explicit is better

@armfazh 

What do you think of where this is heading ? I don't think it really needs to be over-engineered to support multiple instances of Group implementations concurrently, or necessarily require a whole new "group" set of packages (a kind of premature opt). 

This should be a fairly easy to review first step.

The main point is that you would have separate submodule[s] that would easily be excluded by bundler, via not exposing them through the main entry point. And people could choose to take on any (perceived) risk of using the @noble group implementation, while cloudflare gets comfortable with it. 

Note that I have nothing to do with @noble, just want a performant impl of voprf. I've been through migrating from sjcl to elliptic (used by crypto-browserify for ecc) back in the day when I worked for @ripple, as we wanted ed25519 support, and it's clear @noble is the successor to elliptic (which actually doesn't have that great p256 performance ... ). It's interesting to note that @indutny is a sponsor of Paul's work here. Somewhat of a passing of the torch ? 

Thoughts ?

Reply:
@armfazh @chris-wood

What if this was simply scoped to creating the ability to swap out the group, with SCJL preconfigured? i.e. no changes required for consumers of the library.

In any case, it doesn't seem like anyone is actually using this library at present ?

Reply:
At the current proposal, sjcl is preconfigured. ideally, users needs to do the minimal configuration when noble (or other package) is available for use.

Reply:
I guess you could do simply 
`import 'xxxx`` and have it automatically patch voprf-ts 
Is what what you had in mind with `minimal configuration` ?

Reply:
In any case, it doesn't really matter from the POV of voprf-ts, just that it honors any configuration of Oprf.Group ?

Reply:
I am ok with setting the group as 

```ts
import { NobleGroup } from 'xxxx'
Oprf.Group = NobleGroup
```

Reply:
Ok